src/main/java/org/llorllale/cactoos/matchers/MatcherOf.java

```diff
@@ -46,9 +46,8 @@
  *
  * @param <T> Type of object to match
  * @since 0.12
- * @todo #135:30min Remove all constructors except the last one so that
- *  every matcher implemented using MatcherOf take care of properly
- *  describe itself and the mismatch.
+ * @todo #165:30m Rewrite the Matcher's, which constructed from others constructor,
+ *  to use the last constructor of MatcherOf. And delete all constructors except last one.
```

Comment:
@ruca905 I'm confused, didn't you already remove all the constructors except the last one? Please update the todo accordingly...